model/schema/controlledVocabulary.yaml

```diff
@@ -80,3 +73,11 @@ classes:
       name:
         required: true
         identifier: true
+
+  VocabularyTermSynonym:
+    is_a: Synonym
+    description: >-
+      The relationship between a controlled vocabulary term and a synonym.
+    slot_usage:
+      subject:
+        range: VocabularyTerm
```

Comment:
Is this for the synonym of a VocabularyTerm?  If so, VocabularyTerm is the domain, not the range.  Also, I don't see a slot on VocabularyTerm pointing to this.

Reply:
duplicate comment, sorry

Reply:
Yes, this is a synonym for a vocabulary term. The proposed VocabularyTerm entity has a subject slot that points to the VocabularyTerm, rather than a synonyms slot in the VocabularyTerm that points to a Synonym.